certgen/certgen.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+const (
+	caCrt = `
```

Comment:
Sorry I wasn't clear, by generating I meant actually generating, not embedding 3 files, this isn't generating anything - it just writes 3 file content, which has no benefit over checking in files (which we don't want to do)

Anyway,  the build image does have openssl
```
$ docker run -it istio/fortio.build:v7
root@e4d71cc7d777:/go/src/istio.io# which openssl
/usr/bin/openssl
root@e4d71cc7d777:/go/src/istio.io# openssl version
OpenSSL 1.0.2g  1 Mar 2016
```
so just add a make target as I mentioned